P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -815,7 +815,6 @@ target="#CONA"/>).  The tag -->The
         of one or other of the above three classes. Generic elements for state, trait, and event are also defined: <specList>
           <specDesc key="state"/>
           <specDesc key="trait"/>
-          <specDesc key="event" atts="where"/>
```

Comment:
why is `<event>` removed completely?

Reply:
@peterstadler Oops! Thanks for catching that--I was just intending to remove the atts.